scripts/migrate_instruments_to_gcp.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import requests
+from requests.exceptions import HTTPError
+
+# Run this script by doing `pipenv run python scripts/migrate_instruments_to_gcp.py`
+
+# Populate this with the instrument ids you wish to migrate
+instrument_ids = []
+
+print("Beginning migrating instruments")
+
+for instrument_id in instrument_ids:
+    print(f"Migrating instrument [{instrument_id}")
+    # If you're port forwarding, this should remain the same regardless of environment
+    url = f"http://localhost:8002/collection-instrument-api/1.0.2/migrate/{instrument_id}"
```

Comment:
localhost?

Reply:
Yeah, when you run the script you have to set up port forwarding to the collection instrument pod.   It's a one time use script so I didn't put too much into making it configurable.  Just change the port to whatever you need it to be and go from there